creator-node/src/routes/tracks.js

```diff
@@ -67,26 +68,34 @@ router.post(
     })
 
     if (selfTranscode) {
-      await AsyncProcessingQueue.addTrackContentUploadTask({
-        logContext: req.logContext,
-        req: {
-          fileName: req.fileName,
-          fileDir: req.fileDir,
-          fileDestination: req.file.destination,
-          cnodeUserUUID: req.session.cnodeUserUUID
-        }
+      getTracer().startActiveSpan('transcode', async (span) => {
```

Comment:
mind quickly summarizing what this code does?

Reply:
I'm still modifying it a ton - I can add tons of docs and comments on things before I get the PR completely ready

Reply:
sounds great! no worries 🙂